### PR TITLE
Expanding note

### PR DESCRIPTION
**Pull request by get-itips**
Based on experience
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2700

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/organization-assets-library.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ If your organization needs to store and manage files for all your users to use,
 
     > [!NOTE]
     > - For the organization assets library to appear to a user in PowerPoint on the web, the user must be assigned a license to Office 365 E3 or E5. For users who use the PowerPoint desktop app, they also need Microsoft 365 Apps Version 2002 or later. 
+    > - Allow up to 24 hrs for the organization assets library to appear to a user in the desktop apps.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/organization-assets-library.md:39]:
```suggestion
    > - Allow up to 24 hours for the organization assets library to appear to a user in the desktop apps.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - minor suggestion.

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
@kaarins Hello, can I kindly ask this to be reviewed and merged? Thank you

**Comment by get-itips** [comment]
@kaarins Please review and merge, thank you

**Pull request merged**